Content.Server/Entry/EntryPoint.cs

```diff
@@ -79,7 +80,7 @@ public override void Init()
 
             foreach (var callback in TestingCallbacks)
             {
-                var cast = (ServerModuleTestingCallbacks) callback;
+                var cast = (ServerModuleTestingCallbacks)callback;
```

Comment:
unrelated whitespace change
```suggestion
                var cast = (ServerModuleTestingCallbacks) callback;
```